### DISCUSSION TITLE
QoL - Autocompletion of open channel names when adding splits to a tab

### DISCUSSION THREAD

**Discussion by ghost** [Chatterino/chatterino2#5203] [Ideas]
For convenience when selecting a channel to split up the current tab with there should be autocompletion with the current open channel names (like the autocompletion used in Ctrl + K). 

Alternatively a dropdown menu or the ability to drag and drop open tabs to combine.

**Comment by Nerixyz** [comment]
It could also be nice to include the followed channels in the autocompletion (requires `user:read:follows` scope). For the completion, [`QCompleter`](https://doc.qt.io/qt-6/qcompleter.html) can be used.

**Comment by pajlada** [comment]
I want to be a bit careful of autocompletion on input fields like this, it's very easy to get it wrong so it instead hinders the user or makes them accidentally add the wrong split.
I'm not opposed to it or what Nerixyz mentioned, I just want to shout out that this would have to implemented with great care
